requirements-common.txt

```diff
@@ -35,3 +35,4 @@ setuptools>=74.1.1; python_version > '3.11' # Setuptools is used by triton, we n
 einops # Required for Qwen2-VL.
 compressed-tensors == 0.8.0 # required for compressed-tensors
 depyf==0.18.0 # required for profiling and debugging torch.compile
+librosa >= 0.10.2  # required for audio processing including Whisper
```

Comment:
I believe we made librosa optional in the past due to licensing concerns, I can find the issue that started this if needed

We currently have it as a dep through the vllm[audio] extra here https://github.com/vllm-project/vllm/blob/fdea8ec16775e1645620b5ff46b799d60df4624c/setup.py#L633